velox/docs/configs.rst

```diff
@@ -394,6 +394,11 @@ Hive Connector
      - 20000
      - Maximum number of entries in the file handle cache. The value must be non-negative. Zero value
        indicates infinite cache capacity.
+   * - file_handle_cache_enabled
```

Comment:
Can we simply set num_cached_file_handles to zero to disable it instead of having a new config?

Reply:
num_cached_file_handles = 0 has a special meaning of unlimited cache capacity.